internal/config/cgroupmanager/cgroupfsmanager.go

```diff
@@ -0,0 +1,84 @@
+// +build linux
+
+package cgroupmanager
+
+import (
+	"fmt"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/containers/libpod/pkg/cgroups"
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+	"github.com/sirupsen/logrus"
+)
+
+type CgroupfsManager struct{}
```

Comment:
Non blocking and leave it as you prefer: We could make it a string type alias if it has no members and just a name

Reply:
I think I like this way better